assembl/static2/tests/unit/actions/adminActions.spec.js

```diff
@@ -0,0 +1,75 @@
+import * as actions from '../../../js/app/actions/adminActions';
+
+describe('Admin actions', () => {
+  describe('updateSelectedLocale action', () => {
```

Comment:
it is quite simple to write tests for simple actions but you want to ensure that your variables are passed correctly to the reducer